examples/scripts/aml-check-usa.php

```diff
@@ -9,12 +9,12 @@
 use Yoti\Entity\AmlProfile;
 use Yoti\YotiClient;
 
-$postCode = '012345';
+$zipCode = '12345';
```

Comment:
Yes this is not an internal code, it's an example that people will be looking at. They should know what they are passing around. $zipCode is the real term here I think.

Reply:
I do agree with the principle of sticking with postcode everywhere, but if we were going to deviate from this, then it would make sense to be in a standalone `AML Check USA` example, like this is. A compromise could be to call it $postcode, but have an inline comment that this is a Zip Code for US users? 

Incidentally, `postcode` is all one word, rather than `postCode` - http://www.future-perfect.co.uk/grammar-tip/is-it-post-code-or-postcode/

Reply:
US users will know what a postcode is, we don't need to rename it as zip code for their sake.

Reply:
What's the point of all the struggles to give a meaningful name to a variable?